src/quo2/components/tags/context_tag/style.cljs

```diff
@@ -37,21 +17,91 @@
 
 (def audio-tag-icon-color colors/white)
 
-(defn audio-tag-text-color
-  [override-theme]
-  (colors/theme-colors colors/neutral-100 colors/white override-theme))
-
-(def community-tag
-  {:padding-vertical 2
-   :padding-left     2})
+(defn container
+  [theme type size state blur? customization-color]
```

Comment:
Single map argument can also be used here

Reply:
I know, but I prefered to have  a more concise call, also this style function is only used once :thinking: 
 I changed it but still don't like too much the change:

![image](https://github.com/status-im/status-mobile/assets/90291778/0e42024b-7356-457b-b679-1b872aadb07c)